docs/index.html

```diff
@@ -85,13 +95,16 @@ <h1>image edit tools</h1>
         )
     }
 
-    function previewBlob(blob) {
+    function previewBlob(blob, ext) {
         let fr = new FileReader()
         fr.readAsDataURL(new File([blob], "name", {lastModified: 0}))
         fr.onload = function () {
             let img = document.createElement('img')
             let info = document.getElementById('size-info')
-            img.setAttribute('src', String(fr.result))
+            let result = fr.result
+            let srcMime = fr.result.slice(fr.result.indexOf(":") + 1, fr.result.indexOf(";"))
+            result = fr.result.replace(srcMime, "image/" + ext)
+            img.setAttribute('src', String(result))
```

Comment:
```suggestion
            let result = fr.result
            let srcMime = result.slice(result.indexOf(":") + 1, result.indexOf(";"))
            let replaceMime = result.replace(srcMime, "image/" + ext)
            img.setAttribute('src', String(replaceMime))
```

This may look better.